extension/source/Home/Wallet/Wallets/ActionsModal.tsx

```diff
@@ -0,0 +1,74 @@
+import { X } from 'phosphor-react';
+import { useState } from 'react';
+import Modal from 'react-modal';
+import Button from '../../../components/Button';
+import TransactionCard from '../../../Confirm/TransactionCard';
+import { SendTransactionParams } from '../../../types/Rpc';
+
+const ActionsModal = (cell: { getValue: () => SendTransactionParams[] }) => {
+  const [modalIsOpen, setIsOpen] = useState(false);
+
+  const actions = cell.getValue();
+
+  function openModal() {
+    setIsOpen(true);
+  }
+
+  function closeModal() {
+    setIsOpen(false);
+  }
```

Comment:
Non-blocking, but I think it would be better to inline functions like these. Did you wrap these intentionally or were they just more complex before and you kept the wrapper after simplifying them?